iOS/_Stream Temp/ChatMessage+Extensions.swift

```diff
@@ -131,19 +131,9 @@ extension Message: Messageable {
     }
     
     var emotions: [Emotion] {
-        let controller = ChatClient.shared.messageController(for: self)
-        
-        guard let data = controller?.message?.extraData["emotions"],
-              case .array(let JSONObjects) = data else {
-                  return []
-              }
-        
-        let emotions: [Emotion] = JSONObjects.compactMap({ json in
-            guard case .string(let value) = json else { return nil }
-            return Emotion.init(rawValue: value)
-        })
-        
-        return emotions
+        return self.reactionScores.compactMap { dict in
```

Comment:
notice here we return this dict, vs the reaction objects themselves. These should always be on the message where the reactions may have to be loaded.